content/documentation/staging/tutorial/02-install-travels-demo/02-design-travels-demo.adoc

```diff
@@ -0,0 +1,58 @@
+---
+title: Review the demo application
+weight: 22
+---
+:icons: font
+:sectlinks:
+
+The Travels Demo application simulates two business domains organized in different namespaces:
+
+===== Travel Portal namespace
+
+In a first namespace called *travel-portal* there will be deployed several travel shops, where typically users access to search and book flights, hotels, cars or insurances.
+
+There will be several shops to simulate that business of every portal may be different. There will be different characteristics of types of travels, channel (web, mobile), or target (new or existings customers).
+
+These workloads may generate different type of traffic to imitate different real scenarios.
+
+All the portals use a service called _travels_ deployed in the *travel-agency* namespace.
+
+===== Travel Agency namespace
+
+A second namespace called *travel-agency* will host a set of services created to provide quotes for travels.
+
+A main _travels_ service will be the business entry point for the travel agency. It receives a destination city and a user as parameters and it calculates all elements that compose a travel budget: airfares, lodging, car reservation and travel insurances.
+
+There are several services that calculates a separate price and the travels service is responsible to aggregate them in a single response.
+
+Additionally, some users like _registered_ users can have access to special discounts, managed as well by an external service.
+
+Service relations between namespaces can be described in the following diagram:
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/02-02-travels-demo-design.png" title="Travels Demo Design">
+    <img src="/images/tutorial/02-02-travels-demo-design.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+===== Travel Portal and Travel Agency flow
+
+A typical flow consists of the following steps:
+
+. A portal queries the travels service for available destinations.
+. Travels service queries the available hotels and return to the portal shop.
+. A user selects a destination and a type of travel, which may include a _flight_ and/or a _car_, _hotel_ and _insurance_.
+. _Cars_, _Hotels_ and _Flights_ may have available discounts depending of user type.
+
+===== Travel Control namespace
+
+In the Travels Demo it has been introduced a *business dashboard* with two features:
+
+* Allow to change the settings of every travel shop simulator (traffic ratio, device, user and type of travel).
+* Providing a *business* view of the total requests generated from the *travel-portal* namespace to the *travel-agency* services, organized with business criteria as grouped per shop, per type of traffic and per city.
+
```

Comment:
Not sure if I understand your comment here.
It's describing the key features of the *control* app.

Reply:
I am just saying that at this point, because I had already deployed the travels demo, I thought I was being asked to actually bring up the console/business view.  Then I realized it was just telling me about the console and there was no action to take at this point.   I will resolve this conversation and instead add some specific change recommendations.

Reply:
Actually, you can just leave it as is.  I'm not sure why I was confused here.   In the overall context it seems fairly clear.  For some reason I was thinking we needed a line prior to the screenshot like:

`Later in the tutorial we will learn how to bring up the console, it looks like this:`

But I no longer think an addition is really necessary.